docs/api/radio-group.md

```diff
@@ -2,8 +2,6 @@
 title: "ion-radio-group"
 hide_table_of_contents: true
 ---
-import Tabs from '@theme/Tabs';
```

Comment:
Don't forget to get rid of the inline TOC as well 👀

Reply:
Ahh I thought we were only doing that when we added playgrounds, will update.